src/webrtc.ts

```diff
@@ -508,37 +513,67 @@ export class WebrtcExtension extends EventEmitter {
     this.logger.info('sent jingle retract', { sessionId, conversationId: session.propose.conversationId });
   }
 
-  async refreshIceServers (): Promise<any[]> {
+  async refreshIceServers (): Promise<ExtendedRTCIceServer[]> {
+    if (!this.refreshIceServersRetryPromise) {
+      this.refreshIceServersRetryPromise = retryPromise<ExtendedRTCIceServer[]>(
```

Comment:
Converted this to use a `retryPromise` to retry 3 times if this errors for any reason.